src/utils/notifications.ts

```diff
@@ -72,6 +72,23 @@ export const handlePushManagerUnsubscribe = async (hide: () => void) => {
   }
 };
 
+export const blockNotification = async () => {
+  const data = {
+    accountId: 'erditkurteshi.near',
+    endpoint: '/preferences/set',
+    dapp: 'mention',
+    block: true,
+  };
+
+  await fetch(`${notificationsHostName}/preferences/set`, {
```

Comment:
```suggestion
  await fetch(`${notificationsHostName}/${endpoint}`, {
```

And from now we'll have only one configurable method for managing notifications.

Reply:
Correct me if I'm wrong. But If I understood everything properly then you can simply use `manageNotification ` method in components repo